<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleOrderChangesBM</class>
 <widget class="QWidget" name="battleOrderChangesBM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelMode">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Режим</string>
     </property>
     <property name="buddy">
      <cstring>lineEditMode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditMode">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelPosition">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Положение и состояние:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelState">
     <property name="text">
      <string>Состояние</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxManner">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="4">
    <layout class="QFormLayout" name="formLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>143</number>
     </property>
     <property name="bottomMargin">
      <number>24</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTime_1">
       <property name="text">
        <string>Время</string>
       </property>
       <property name="buddy">
        <cstring>lineEditTimeState</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTimeState">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTime_2">
       <property name="text">
        <string>Время</string>
       </property>
       <property name="buddy">
        <cstring>lineEditTimeCoordinates</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditTimeCoordinates">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonPickCoordinates">
       <property name="text">
        <string>Съем координат</string>
       </property>
       <property name="autoRepeatDelay">
        <number>285</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelCoordinates">
     <property name="text">
      <string>Координаты</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditCoordinates"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelAzimuth">
     <property name="text">
      <string>Азимут</string>
     </property>
     <property name="buddy">
      <cstring>lineEditAzimuth</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditAzimuth"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelDataRockets_1">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Данные о ракетах, контейнер №1:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>№ </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labeDataType_1">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string> Тип БЧ       </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_1"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_3"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_5"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_7"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelDataTime_1">
       <property name="text">
        <string>Время данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelDataState_1">
       <property name="text">
        <string>Состояние/контроль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_1"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_3"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_5"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_7"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="labelDataRockets_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Данные о ракетах, контейнер №2:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>№ </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="labeDataType_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string> Тип БЧ       </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_2"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_4"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_6"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType_8"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="8" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="labelDataTime_2">
       <property name="text">
        <string>Время данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="labelDataState_2">
       <property name="text">
        <string>Состояние/контроль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_2"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_4"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_6"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxState_8"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labelData">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Данные об имуществые и обеспеченности:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidgetData">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>163</number>
     </attribute>
     <column>
      <property name="text">
       <string>Имущество</string>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество</string>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Место хранения</string>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <zorder>labelMode</zorder>
  <zorder>lineEditMode</zorder>
  <zorder>labelPosition</zorder>
  <zorder>labelState</zorder>
  <zorder>labelCoordinates</zorder>
  <zorder>labelAzimuth</zorder>
  <zorder>labelDataRockets_1</zorder>
  <zorder>labelDataRockets_2</zorder>
  <zorder>labelData</zorder>
  <zorder>tableWidgetData</zorder>
  <zorder>comboBoxManner</zorder>
  <zorder>lineEditAzimuth</zorder>
  <zorder></zorder>
  <zorder>lineEditCoordinates</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
